lib/strava/api/ratelimit_status.rb

```diff
@@ -0,0 +1,59 @@
+module Strava
+  module Api
+    class RatelimitStatus
+      attr_reader :headers
+
+      def initialize(headers)
+        @headers = headers
+      end
+
+      def ratelimit_included?
+        puts !!@headers['x-ratelimit-limit']
+      end
+
+      def ratelimit_limit
+        @headers['x-ratelimit-limit']
+      end
+
+      def ratelimit_limit_usage
+        @headers['x-ratelimit-usage']
+      end
+
+      def ratelimit_fiveteen_minutes
+        ratelimit_limit.split(',').first.to_i
+      rescue NoMethodError
```

Comment:
Don't rely on the exceptions. This should be something like

`ratelimit_limit.split(',').first.to_i if ratelimit?`

Reply:
using ternary operator now